src/eventhandlergui.cxx

```diff
@@ -251,7 +251,8 @@ void handleGuiEvents()
 						gui->getTrack(ev.track)->getRadialStatus()->recording( true );
 					else if ( ev.state == GridLogic::STATE_EMPTY )
 						// reset clip name if clip gets cleared
-						gui->getTrack(ev.track)->getClipSelector()->clips[ ev.scene ].setName(""); 
+						gui->getTrack(ev.track)->getClipSelector()->clips[ ev.scene ].setName("");
+						gui->getTrack(ev.track)->getRadialStatus()->recording(false);
```

Comment:
You missed some { and } brackets - this ain't python with whitespace de-limited `if` conditions ;)